rom/dev/tools/test-rt/src/main.rs

```diff
@@ -25,6 +25,8 @@ mod print;
 #[cfg(feature = "std")]
 pub fn main() {}
 
+// Dummy RO data to max out FW image size.
+static PAD: [u32; 27000] = [0xDEADBEEFu32; 27000];
```

Comment:
How do we know that 128k - size_of(fmc binary) - size_of(rt_binary) = 27000?

Reply:
It's not exactly 128K; it's a ballpark number close to 128K.